vortex-duckdb/src/exporter/struct_.rs

```diff
@@ -52,6 +55,35 @@ impl ColumnExporter for StructExporter {
     }
 }
 
+pub(crate) fn new_vector_exporter(
+    array: StructArray,
+    cache: &ConversionCache,
+    session: &VortexSession,
+) -> VortexResult<Box<dyn ColumnExporter>> {
+    let validity = array.validity_mask();
```

Comment:
See this is weird.. validity_mask is deprecated since it's old "canonical", you should use Validity + execute_mask